tests/topotests/bgp_multiview_topo1/peer2/exa-send.py

```diff
@@ -23,7 +23,7 @@
     stdout.flush()
 
 # Announce 1 overlapping route per peer
-stdout.write("announce route 10.0.1.0/24 next-hop 172.16.1.%i\n" % peer)
+stdout.write("announce route 10.0.1.0/24 med %i next-hop 172.16.1.%i\n" % (peer, peer))
```

Comment:
the med is the peer?  What does this turn into?

Reply:
Seems this is just an integer to distinguish peer/med equally.